impeller/entity/contents/solid_rrect_blur_contents.cc

```diff
@@ -16,6 +16,13 @@
 
 namespace impeller {
 
+namespace {
+// Generous padding to avoid bugs.
```

Comment:
Nit: be more specific here, generous padding to avoid hard edges on the blur? And perhaps a note that the blurring makes the rrect "bigger" than the initial geometry.

Reply:
done